R/as_rtf_pageby.R

```diff
@@ -139,10 +139,18 @@ as_rtf_pageby <- function(tbl) {
 
   ## Remove repeated records if group_by is not null
   if (!is.null(group_by)) {
-    cell_tbl <- rtf_group_by_enhance(cell_tbl,
-      group_by = group_by,
+    tmp <- cell_tbl
+    tmp$`.pageby` <- pageby$id
+    tmp$`.order` <- 1:nrow(tmp)
```

Comment:
`1:nrow(tmp)` is likely to be wrong in the empty edge case. Use `seq_len(nrow(tmp))` instead.